src/StorageApiToken.php

```diff
@@ -95,13 +95,18 @@ public function getPermissions(): array
     {
         return array_filter(
             array_keys(
-                array_filter($this->tokenInfo, function ($value) {
+                array_filter($this->tokenInfo, function (mixed $value): bool {
                     return $value === true;
                 }),
             ),
-            function (string $value) {
-                return preg_match('/^can[a-z]+$/ui', $value);
+            function (string $value): bool {
+                return (bool) preg_match('/^can[a-z]+$/ui', $value);
             },
         );
     }
+
+    public function isAdminToken(): bool
```

Comment:
Nechtelo se mi to nazyvat `isMasterToken` a porovnavat to podle `isMasterToken` property.

- `isMasterToken` vzniknul na samem zacatku, kdyz jsme jeste nemeli usery
- isMasterToken=true jen pokud token patri adminovi => takhle je to ok
- `isMasterToken` jsme v roce 2020 oznacili v kodu KBC jako deprecated, tak je vidina toho ze ta property v roce 2150 zmizi